packages/devtools_app/test/chart_test.dart

```diff
@@ -57,7 +57,7 @@ void main() {
             symbol: ChartSymbol.disc,
             diameter: 1.5,
           ),
-          name: 'External',
+          name: 'Flutter engine/rendering',
```

Comment:
Same here, and other places where each trace is created, should reference the one const name.